src/constants.js

```diff
@@ -36,4 +36,4 @@ export const TWITTER_BLUE_COLOR = '#4199d4';
 export const GITHUB_GRAY_COLOR = '#333';
 export const YOUTUBE_RED_COLOR = '#c4302b';
 
-export const MAX_WIDTH = '1200px';
+export const MAX_WIDTH = '2400px';
```

Comment:
What screens does this fit?

Reply:
I kinda eye-balled it... wide monitors... not quite TVs... although that'd be good to do as well

Reply:
I would dive a little deeper on this, I want to understand the use case for screens over 1200px.